operations/power_management/Power_On_Compute_Cabinets.md

```diff
@@ -41,6 +41,21 @@ power-on command from Cray System Management \(CSM\) software.
 
    ![PSU Status LEDs](../../img/operations/PSU_Status.svg)
 
+1. (`ncn-m001#`) Unsuspend the `hms-discovery cronjob` to re-enable the `hms-discovery` job.
+
+   ```bash
+   kubectl -n services patch cronjobs hms-discovery -p '{"spec" : {"suspend" : false }}'
+   ```
+
+   Example output.
+
+   `ACTIVE` = `1` and `SUSPEND` = `False` in the output indicates that the job has been unsuspended:
+
+   ```text
+   NAME             SCHEDULE      SUSPEND   ACTIVE   LAST   SCHEDULE  AGE
+   hms-discovery    */3 * * * *   False       1      41s              33d
+   ```
```

Comment:
The explanation isn't exactly right and might be misleading about what to expect. It should not be required for `ACTIVE` to be `1`. Spacing in the example output is also off.

````suggestion
   Example output:

   ```text
   NAME             SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE  AGE
   hms-discovery    */3 * * * *   False     1        41s            33d
   ```

   A value of `False` in the `SUSPEND` column indicates that the cronjob is no longer suspended. A
   value of `1` in the `Active` column indicates that a Kubernetes job is currently running for the
   cronjob.
````

Reply:
fixed